contracts/payloads/IGP108/PayloadIGP108.sol

```diff
@@ -0,0 +1,581 @@
+            }
+        }
+        {
+            // Vault ID 16: Weeth/Wsteth - Max Borrow to $300M
+            address weETH_wstETH_VAULT = getVaultAddress(16);
+
+            BorrowProtocolConfig memory protocolConfig_ = BorrowProtocolConfig({
+                protocol: weETH_wstETH_VAULT,
+                borrowToken: wstETH_ADDRESS,
+                expandPercent: 50 * 1e2, // 50%
+                expandDuration: 6 hours, // 6 hours
+                baseBorrowLimitInUSD: 20_000_000, // $20M base limit
+                maxBorrowLimitInUSD: 300_000_000 // $300M max limit
+            });
+
+            setBorrowProtocolLimits(protocolConfig_);
+        }
+        {
+            // Vault IDs 31,32,33: ETH/cbBTC Vaults - Max Borrow to $70M
+            {
+                address ETH_cbBTC_VAULT_31 = getVaultAddress(31);
+
+                BorrowProtocolConfig
+                    memory protocolConfig_ = BorrowProtocolConfig({
+                        protocol: ETH_cbBTC_VAULT_31,
+                        borrowToken: cbBTC_ADDRESS,
+                        expandPercent: 50 * 1e2, // 50%
+                        expandDuration: 6 hours, // 6 hours
+                        baseBorrowLimitInUSD: 15_000_000, // $15M base limit
+                        maxBorrowLimitInUSD: 70_000_000 // $70M max limit
+                    });
+
+                setBorrowProtocolLimits(protocolConfig_);
+            }
+            {
+                address ETH_cbBTC_VAULT_32 = getVaultAddress(32);
+
+                BorrowProtocolConfig
+                    memory protocolConfig_ = BorrowProtocolConfig({
+                        protocol: ETH_cbBTC_VAULT_32,
+                        borrowToken: cbBTC_ADDRESS,
+                        expandPercent: 50 * 1e2, // 50%
+                        expandDuration: 6 hours, // 6 hours
+                        baseBorrowLimitInUSD: 15_000_000, // $15M base limit
+                        maxBorrowLimitInUSD: 70_000_000 // $70M max limit
+                    });
+
+                setBorrowProtocolLimits(protocolConfig_);
+            }
+            {
+                address ETH_cbBTC_VAULT_33 = getVaultAddress(33);
+
+                BorrowProtocolConfig
+                    memory protocolConfig_ = BorrowProtocolConfig({
+                        protocol: ETH_cbBTC_VAULT_33,
+                        borrowToken: cbBTC_ADDRESS,
+                        expandPercent: 50 * 1e2, // 50%
+                        expandDuration: 6 hours, // 6 hours
+                        baseBorrowLimitInUSD: 15_000_000, // $15M base limit
+                        maxBorrowLimitInUSD: 70_000_000 // $70M max limit
+                    });
+
+                setBorrowProtocolLimits(protocolConfig_);
+            }
+        }
+    }
+
+    // @notice Action 5: Absorb Dust Debt for wstETH-WBTC
+    function action5() internal isActionSkippable(5) {
+    //     address wstETH_WBTC_VAULT = getVaultAddress(25);
```

Comment:
@xma8lu This part is incorrect; 1 to 18 NFT IDs might not be from this vault itself. So, if there are 18 positions, then they can have different IDs on the EVM side. So, first we need to know all the NFT IDs and then hardcode them